terraform/shell.nix

```diff
@@ -0,0 +1,84 @@
+let
+  sources = import ./nix/sources.nix;
+  overlay = _: pkgs: {
```

Comment:
🧹

Reply:
Eh, I kind of want to keep it in case there are further additions.